cloud/jenkins/pgo_operator_aws_openshift-latest.groovy

```diff
@@ -0,0 +1,317 @@
+void pushArtifactFile(String FILE_NAME) {
+    echo "Push $FILE_NAME file to S3!"
```

Comment:
I believe we can put some fancy "latest" logic into existing openshift job. 
The same is true for gke.

If you have any objections please comment

Reply:
Yes, we can. But @hors said that we should duplicate the jobs